addons/explosives/stringtable.xml

```diff
@@ -508,26 +508,31 @@
         <Key ID="STR_ACE_Explosive_Module_DisplayName">
             <English>Explosive System</English>
             <Polish>System ładunków wybuchowych</Polish>
+            <German>Sprengstoffsystem</German>
         </Key>
         <Key ID="STR_ACE_Explosive_RequireSpecialist_DisplayName">
             <English>Require specialists?</English>
             <Polish>Wymagaj specjalistów?</Polish>
+            <German>Benötigt Experten?</German>
         </Key>
         <Key ID="STR_ACE_Explosive_RequireSpecialist_Description">
             <English>Require explosive specialists to disable explosives? Default: No</English>
             <Polish>Wymagać saperów do rozbrajania ładunków wybuchowych? Domyślnie: Nie</Polish>
+            <German>Benötige Sprengstoffexperte um Sprengladungen zu entschärfen? Standard: Nein</German>
         </Key>
         <Key ID="STR_ACE_Explosive_PunishNonSpecialists_DisplayName">
             <English>Punish non-specialists?</English>
             <Polish>Karaj nie-specjalistów?</Polish>
+            <German>Bestrafe Nicht-Experten?</German>
```

Comment:
I don't think that you can make that sound good in german without using more words. I'd leave it as is.